build-scripts/build.sh

```diff
@@ -0,0 +1,34 @@
+#!/bin/bash
```

Comment:
I think this build script has to be added as an [npm script](https://docs.npmjs.com/misc/scripts) in the right stage, to ensure it is run automatically when the aws-sdk version is changed, at least before publishing.

`prepare` might be the right one - we need to run this on `npm install` in this project, in case a new aws-sdk version is pulled in (note: this is assuming `devDependencies` that I mention in another comment, as the main control for the version), and of course before publishing.

Just so that it all fits into the usual `npm` commands.